src/clarity-angular/nav/_nav.clarity.scss

```diff
@@ -49,8 +49,7 @@ $clr-active-nav-link-shadow: 0 -3px 0 $action-blues-dark-midtone inset;
             }
 
             &:hover,
-            &.active,
-            &:focus {
```

Comment:
cool. since the focus state is changing we need to make sure that @yenmade knows about this and approves it.